hack/run-lint.sh

```diff
@@ -28,4 +29,19 @@ then
 fi
 
 # configured by .golangci.yml
-exec env GO111MODULE=on "$gopath/bin/golangci-lint" run
```

Comment:
There has been a discussion that we want to limit the scope of env variables as much as possible. Could you therefore inline the `GO111MODULE` env variable where necessary?

Reply:
Do you have the link of this discussion so that I can read it too?

I will revert it to stick to that discussion conclusion.

Reply:
reverted.

Reply:
It's here https://github.com/kubernetes-sigs/krew/pull/243#discussion_r300064338.